### PR TITLE
BOM-1044 Django22 support

### PR DESCRIPTION
**Pull request by Ayub-Khan**
-Updated travis file
-updated requirements
-updated tox test envs

### CODE REVIEW

**Comment by Ayub-Khan** [comment]
this is dependent of xblock-sdk django22 support.

**Comment by Ayub-Khan** [comment]
this might require those changes in Xblock(https://github.com/edx/XBlock/pull/438). Looking issue in more detail.

File: requirements/base.in [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Core requirements for using this package
 
+django-nose
```
Comment (jmbowman) [inline requirements/base.in]: edx-platform depends on xblock-utils and currently doesn't need this; does this really need to be a core dependency of the package, or just a testing dependency?

**Comment by Ayub-Khan** [comment]
> One suggestion, otherwise looks good. Maybe do a pytest upgrade in a separate PR to make sure we've caught all the deprecation warnings?

sure. i will create a new one pr for that.

File: requirements/base.in (hunk 1 above)
Comment (Ayub-Khan) [reply requirements/base.in]: It seems it was no longer being used. Removed it.

**Pull request merged**